tools/labs.cmake

```diff
@@ -51,13 +51,8 @@ endif()
 set(CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS} ${CMAKE_CXX_FLAGS}")
 
 # https://github.com/google/benchmark
-if (WIN32 AND EXISTS "C:/Program Files (x86)/benchmark")
-  set(BENCHMARK_FOLDER "C:/Program Files (x86)/benchmark")
-  find_library(BENCHMARK_LIBRARY NAMES benchmark PATHS "${BENCHMARK_FOLDER}" REQUIRED)
-else()
-  set(BENCHMARK_FOLDER "${CMAKE_CURRENT_LIST_DIR}/benchmark")
-  find_library(BENCHMARK_LIBRARY NAMES benchmark PATHS "${BENCHMARK_FOLDER}/build/src/Release" "${BENCHMARK_FOLDER}/build/src" REQUIRED)
-endif()
+find_package(benchmark REQUIRED)
```

Comment:
`find_package(benchmark PATHS "${CMAKE_CURRENT_LIST_DIR}/benchmark/build" REQUIRED)`
A possibility to use a local benchmark library can be useful.

Reply:
Fixed